recipes/recipes_schubar.json

```diff
@@ -841,6 +841,30 @@
         "ice": "cubed",
         "prep": "build",
         "garnish": "Flag of lemon peel and raspberries"
+    },
+
+    "Cayuga White": {
```

Comment:
Recipes here are generic, so this should really be a single entry of "wine".
Then in your ingredients file you'd make sure that the category listed is Wine, then Ingredient as white wine, then the specific bottle and type for Kind